full-service/src/service/models/tx_proposal.rs

```diff
@@ -81,7 +81,7 @@ impl UnsignedTxProposal {
 
         let signer = LocalRingSigner::from(account_key);
         let mut rng = rand::thread_rng();
-        let tx = self.unsigned_tx.sign(&signer, &mut rng)?;
+        let tx = self.unsigned_tx.sign(&signer, None, &mut rng)?;
```

Comment:
There should be a way for non-off-line full-service to pass the current fee map here. This is part of a TOB audit. See the TOB-5 PRs: https://github.com/mobilecoinfoundation/mobilecoin/pulls?q=is%3Apr+TOB-5